scan/src/subscriptions/ReportSub.re

```diff
@@ -10,6 +10,7 @@ module ValidatorReport = {
   };
 
   type report_details_t = {
+    dataSourceID: int,
```

Comment:
Shouldn't this be `ID.DataSource.t`? @evilpeach

Reply:
Yes